src/components/PoolInfoCard.tsx

```diff
@@ -61,73 +69,96 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
-          <span className="value">{formattedData.swapFee}</span>
-        </div>
-        <div className="infoItem">
+      <InfoItem mt={3}>
+        <Typography>{`${t("status")}:`}</Typography>
+        <Typography variant="subtitle1">{`${
+          data?.isPaused ? t("paused") : t("active")
+        }`}</Typography>
+      </InfoItem>
+      <InfoItem>
+        <Typography>{`${t("totalLocked")}:`}</Typography>
+        <Typography variant="subtitle1">{`$${formattedData.reserve}`}</Typography>
+      </InfoItem>
+      {/* <InfoItem>
+        <Typography>{t("dailyVolume") + ": "}</Typography>
+        <Typography variant="subtitle1">{formattedData.volume}</Typography>
+      </InfoItem> */}
+
+      <Box display="flex" mt={3} flexWrap="wrap">
+        <InfoItem>
+          <Typography>{`${t("fee")}:`}</Typography>
+          <Typography variant="subtitle1">{formattedData.swapFee}</Typography>
+        </InfoItem>
+        <InfoItem>
+          <Typography>{`${t("virtualPrice")}:`}</Typography>
+          <Typography variant="subtitle1">
+            {formattedData.virtualPrice}
+          </Typography>
+        </InfoItem>
+        <InfoItem>
           <Tooltip
             title={<React.Fragment>{t("aParameterTooltip")}</React.Fragment>}
           >
-            <span className="label bold underline">{`${t(
-              "aParameter",
-            )}:`}</span>
+            <Typography
+              sx={{ cursor: "help", borderBottom: "1px dotted" }}
+            >{`${t("aParameter")}`}</Typography>
           </Tooltip>
-          <span className="value">{formattedData.aParameter}</span>
-        </div>
-        <div className="infoItem">
-          <span className="label bold">{`${t("virtualPrice")}:`}</span>
-          <span className="value">{formattedData.virtualPrice}</span>
-        </div>
-        <div className="infoItem">
-          <span className="label bold">{`${t("utilization")}:`}</span>
-          <span className="value">{formattedData.utilization}</span>
-        </div>
-        <div className="infoItem">
-          <span className="label bold">{`${t("totalLocked")}:`}</span>
-          <span className="value">{`$${formattedData.reserve}`}</span>
-        </div>
-        <div className="twoColumn">
-          <div className="infoItem">
-            <span className="label bold">{`${t("adminFee")}:`}</span>
-            <span className="value">{formattedData.adminFee}</span>
-          </div>
-          {/* <div className="infoItem">
-            <span className="label bold">{t("dailyVolume") + ": "}</span>
-            <span className="value">{formattedData.volume}</span>
-          </div> */}
-        </div>
-      </div>
-      <Divider />
-      <div className="bottom">
-        <h4>{t("currencyReserves")}</h4>
-        <span>{`$${formattedData.reserve} ${t("inTotal")}`}</span>
-        <div className="tokenList">
+          <Typography variant="subtitle1">
+            :{formattedData.aParameter}
```

Comment:
Why is it for this occurrence the colon is where the value is? There also needs to be a space. Can you move it to the above field?

Reply:
Because tooltop is wrapping the colon as well.